src/modules/LanItems.rb

```diff
@@ -1280,17 +1280,17 @@ def ip_overview(ip)
       when /DHCP/
         bullets << format("%s %s", _("IP address assigned using"), ip)
       else
-        prefixlen = NetworkInterfaces.Current["PREFIXLEN"]
-        if prefixlen
+        prefixlen = LanItems.prefix
```

Comment:
it looks a bit inconsistent. Is it prefix or length of prefix? variable is len, but method is only prefix